eng/common/TestResources/New-TestResources.ps1

```diff
@@ -92,8 +92,13 @@ param (
     [Parameter()]
     [switch] $SuppressVsoCommands = ($null -eq $env:SYSTEM_TEAMPROJECTID),
 
+    # Default behavior is to use -UserAuth
     [Parameter()]
-    [switch] $UserAuth,
+    [switch] $ServicePrincipalAuth,
+
+    [Parameter()]
+    [switch] $FederatedAuth,
```

Comment:
Do we want to throw this under the Provisioner parameter set?

Reply:
If this was placed in a parameter set then it'll require that the other mandatory parameters be specified. In that case, one will always have to specify the provisioner parameters. I think this should be outside the Provisioner param set.